src/partition/repart.c

```diff
@@ -1316,6 +1320,41 @@ static int config_parse_size4096(
         return 0;
 }
 
+static int config_parse_block_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint64_t *blksz = data;
```

Comment:
= ASSERT_PTR(data)

Reply:
I followed what was in `config_parse_size4096()` above. If you insist I'll update it otherwise it has an assert a few lines down just like the other function does.